src/main/java/cms/gov/madie/measure/models/Measure.java

```diff
@@ -61,6 +61,7 @@
       message = "A measure name must contain at least one letter.")
   private String measureName;
 
+  private Boolean active = true;
```

Comment:
Could this be a lower-case `boolean` instead?

Reply:
When this is changed to lowercase the measure.getActive() method is no longer recognized. 
I'm unsure how to handle that